include/bluetooth/bluetooth.h

```diff
@@ -1116,14 +1118,12 @@ uint8_t bt_le_per_adv_sync_get_index(struct bt_le_per_adv_sync *per_adv_sync);
  * to periodic advertising reports from an advertiser. Scan shall either be
  * disabled or extended scan shall be enabled.
  *
- * @param[in] param     Periodic advertising sync parameters.
- * @param[in] cb        Periodic advertising callbacks.
- * @param[out] out_sync Periodic advertising sync object on.
+ * @param[in]  param     Periodic advertising sync parameters.
+ * @param[out] out_sync  Periodic advertising sync object on.
  *
  * @return Zero on success or (negative) error code otherwise.
  */
 int bt_le_per_adv_sync_create(const struct bt_le_per_adv_sync_param *param,
-			      const struct bt_le_per_adv_sync_cb *cb,
```

Comment:
Just a question, would it have made sense to move it to bt_le_per_adv_start instead?

Reply:
Can you elaborate? 

As in moving the callbacks to the start instead? If so, I guess we could do that (similar to the scan callbacks I guess), but we'll still need the separate callback register function in order to support PAST.

Reply:
ok then no. the scan callbacks are just for backwards compebility.